src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredMen = people.filter(man => {
+    const isMan = man.sex === 'm';
+
+    return (century !== undefined)
```

Comment:
```suggestion
    return (!century)
```

or even better reverse condition